src/WebJobs.Script/Workers/Http/DefaultHttpWorkerService.cs

```diff
@@ -47,6 +54,9 @@ internal DefaultHttpWorkerService(HttpClient httpClient, IOptions<HttpWorkerOpti
                 // Set 1 minute greater than FunctionTimeout to ensure invoction failure due to timeout is raised before httpClient raises operation cancelled exception
                 _httpClient.Timeout = scriptHostOptions.Value.FunctionTimeout.Value.Add(TimeSpan.FromMinutes(1));
             }
+
+            _destinationPrefix = new UriBuilder(WorkerConstants.HttpScheme, WorkerConstants.HostName, _httpWorkerOptions.Port).Uri;
```

Comment:
Is it a valid scenario where a customer configures their web app to only accept HTTPS traffic, but this code affects request forwarding because it explicitly uses HTTP? Should we allow the customer to specify or override the scheme?

Reply:
I think your scenario could be valid - @fabiocav, is this an issue customers run into today? I borrowed from the existing pattern (for the `EnableForwardingHttpRequest` prop) in `DefaultHttpWorkerService` (see [here](https://github.com/Azure/azure-functions-host/blob/265424c38d4f1aa453121735ae69fa8423acc290/src/WebJobs.Script/Workers/Http/DefaultHttpWorkerService.cs#L291)) which only uses HTTP.

Reply:
Discussed with @fabiocav offline, this is outside of the scope of the current changes, but it is a valid comment and may be something we want to address in a future update.